lib/pfconfig/namespaces/resource/CaptivePortal.pm

```diff
@@ -0,0 +1,83 @@
+package pfconfig::namespaces::resource::CaptivePortal;
+
+=head1 NAME
+
+pfconfig::namespaces::resource::CaptivePortal
+
+=cut
+
+=head1 DESCRIPTION
+
+pfconfig::namespaces::resource::CaptivePortal
+
+=cut
+
+use strict;
+use warnings;
+use pf::file_paths;
+use POSIX;
+
+use base 'pfconfig::namespaces::resource';
+
+sub init {
+    my ($self) = @_;
+    $self->{config} = $self->{cache}->get_cache('config::Pf');
+}
+
+sub build {
+    my ($self) = @_;
+    my %Config = %{$self->{config}};
+    # CAPTIVE-PORTAL RELATED
+    # Captive Portal constants
+    my %CAPTIVE_PORTAL = (
+        "NET_DETECT_INITIAL_DELAY" => floor($Config{'trapping'}{'redirtimer'} / 4),
```

Comment:
I know this isn't your code, but this part is not ideal.
Shouldn't this be some reasonable default like 3-5 seconds?
The way it's now means that if you set redirtimer to some high value for crappy dhcp clients you end up making it longer for everyone.